### PR TITLE
New landing page

### PR DESCRIPTION
**Pull request by PeterBreen**
Creates a new landing page for DemocracyLab, aka 'homepage' for the organization; giving an overview of who we are, what we do, and how to get involved.

Ready for review, still has some outstanding questions before launch:

### Product Team review:
-  add a citation/source link to external testimonials (i.e. Mayor's twitter) - y/n?

### Dev team fixes before launch
- [x] host user avatars for Testimonials on CDN so links won't change, update Testimonials.jsx
- [x] set this page as "homepage" for both clicking on the logo to return Home, default page shown to anyone who types/clicks `https://democracylab.org` instead of Find Projects, etc.
- [x] remove SplashScreen from Featured Projects page
- [x] add hero image rotation to TopSplash on Landing page, making sure to remove the mid/bottom splash images to avoid duplication
- [x] update Partners section with current DL partners
- [x] fix key warning in Partners map()
- [x] fix internal site navigation URL links in button hrefs
- [x] pass in splash screen buttons as components, not as a string which matches to an if/else block
- [x] center testimonial avatar photo on mobile (without breaking on desktop)
- [x] remove extra semicolon in the sponsor footer render (probably not this PR's error but I can't unsee it)
- [x] use correct recruiting volunteers 3rd SVG
- [x] remove Login before PartnerWithUs page link in "Partner With Us To Organize Your Next Hackathon" section
- [x] change URL to read `section=Home` instead of `section=Landing` (and ideally, just show `https://democracylab.org` without any trailing path information)

#### mobile
- [x] minimum/fixed height on project card logo so ones without 3 subinfo lines aren't tiny
- [x] spacing on right side of project card sub-info section (e.g. url) so it doesn't clip the right side of the card (tablet too? check all sizes)
- [x] space between logos in Partners section (2 per row mobile, up to 4/5 per desktop)

### bonus fixes
- get a version of WTIA's logo without the extra whitespace on top and bottom so we can add it to all logos with CSS, otherwise we'll need a special class for it
- redo 501Commons logo to not have that grey line on the right edge (sorry, my mistake here)

### CODE REVIEW

File: common/components/componentsBySection/FindProjects/SplashScreen.jsx [hunk 1]
```diff
@@ -29,23 +42,63 @@ class SplashScreen extends React.PureComponent<Props> {
...
+  } else if (sec === "landingtop") {
+      return (
+        <React.Fragment>
+          <Button variant="primary" className="LandingController-topsplash-btn" href={url.sectionOrLogIn(Section.FindProjects)}>Get Started</Button>
+        </React.Fragment>
+      )
+    } else if (sec === "landingmid") {
+        return (
+          <React.Fragment>
+            <Button variant="primary" className="LandingController-midsplash-btn SplashScreen-create-project-btn" href="/index/section?=AboutUs">Learn More</Button>
```
Comment (marlonkeating) [inline common/components/componentsBySection/FindProjects/SplashScreen.jsx]:
```suggestion
            <Button variant="primary" className="LandingController-midsplash-btn SplashScreen-create-project-btn" href={url.section(Section.AboutUs)}>Learn More</Button>
```
Wrong URL

File: common/components/controllers/LandingController.jsx [hunk 2]
```diff
@@ -1,58 +1,167 @@
...
+    }
+
+    _renderPartnerSection() {
+      return (
+        <div className="LandingController-partner-section" style={cdn.bgImage('OurVisionBGoverlay.jpg')}>
+          <div className="PartnerSection text-center">
+            <h2>Partner With Us to Organize your Next Hackathon</h2>
+            <p>DemocracyLab is the leading organizer for Tech-for-Good Hackathons.</p>
+            <p>Let us help your company, non-profit or group organize your next Hackathon.</p>
+            <Button variant="outline-dark" href={url.sectionOrLogIn(Section.PartnerWithUs)}>
```
Comment (marlonkeating) [inline common/components/controllers/LandingController.jsx]:
```suggestion
            <Button variant="outline-dark" href={url.section(Section.PartnerWithUs)}>
```

File: common/components/componentsBySection/Landing/RecentProjectsSection.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  render(): React$Node {
+    return (
+      <div className="RecentProjects">
+        <h2 className="RecentProjects-title">Featured Projects</h2>
+        <div className="RecentProjects-cards">
+          {this._renderCards()}
+        </div>
+        <div className="RecentProjects-button">
+          <Button className="RecentProjects-all" href="/index/?section=FindProjects&showSplash=0">See All Projects</Button>
```
Comment (marlonkeating) [inline common/components/componentsBySection/Landing/RecentProjectsSection.jsx]:
```suggestion
          <Button className="RecentProjects-all" href={url.section(Sections.FindProjects)}>See All Projects</Button>
```

File: common/components/componentsBySection/Landing/Partners.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    link: "https://www.netsquared.org/"
+  },
+  {
+    name: "Seattle Community Technology Advisory Board",
+    logo: "https://d1agxr2dqkgkuy.cloudfront.net/img/communitytech.jpeg",
+    link: "http://www.seattle.gov/community-technology-advisory-board"
+  }
+]
+
+/*
```
[the remark this answers is not in the document]
Comment (PeterBreen) [reply common/components/componentsBySection/Landing/Partners.jsx]: No, removed.

File: common/components/componentsBySection/Landing/Testimonials.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    name: "Jenny Durkan",
+    avatar: "",
+    role: "Mayor",
+    project: "City of Seattle",
+    text: "Seattle is home to some of the most innovative thinkers and technologists anywhere in the world who want to use their talent to address social challenges. It’s why we formed an Innovation Advisory Council, and it’s why groups like DemocracyLab exist.",
+    source: ""
+  },
+]
+
+/*
```
[the remark this answers is not in the document]
Comment (PeterBreen) [reply common/components/componentsBySection/Landing/Testimonials.jsx]: As above, removed!

**Pull request merged**